### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Rotzbua**
* Update dependencies
* Fix variables `var`, `let`, `const` to avoid eslint errors

### CODE REVIEW

**Comment by gustav87** [comment]
Hey, thanks for the contribution, are you on the W3C Discord?

File: package.json [hunk 1]
```diff
@@ -13,38 +13,39 @@
...
   "devDependencies": {
-    "@types/chart.js": "^2.9.35",
+    "@types/chart.js": "^2.9.37",
     "@types/chartjs-plugin-annotation": "^0.5.2",
     "@types/debounce": "^1.2.1",
-    "@types/google-spreadsheet": "^3.1.5",
-    "@types/lodash": "^4.14.178",
+    "@types/google-spreadsheet": "^3.3.0",
+    "@types/lodash": "^4.14.191",
+    "@types/node": "^18.11.18",
```
Comment (gustav87) [inline package.json:48]: Why do we need this?

Comment (Rotzbua) [reply package.json:48]: Recommended for `ts-node`

**Comment by Rotzbua** [comment]
> Hey, thanks for the contribution, are you on the W3C Discord?

Hey. I'm not at the W3C discord. Ironically, do not even play W3C. Just watch Creepjack and do some coding on the side. 😄

**Comment by gustav87** [comment]
> > Hey, thanks for the contribution, are you on the W3C Discord?
> 
> Hey. I'm not at the W3C discord. Ironically, do not even play W3C. Just watch Creepjack and do some coding on the side. smile

Oh okay! You're welcome to join us at https://discord.gg/nx5MaQzy if you feel like it :)

**Pull request merged**